src/main/java/mate/academy/spring/controller/InjectController.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.controller;
+
+import java.util.Set;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.AuthenticationService;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/inject")
+public class InjectController {
```

Comment:
JFYI it's better not to create this class as a controller and use PostConstruct, because every time you will need to send a request to this controller to add data to DB manually

Reply:
I couldn't figure out where this annotation lays and which dependencies in pom xml should I add to get it